portfolio/src/main/webapp/script.js

```diff
@@ -68,13 +68,24 @@ async function getCommentsFromServlet() {
   const comments = document.getElementById('comments-container');
   comments.innerHTML = '';
   commentData.forEach(comment => {
-    comments.appendChild(createParagraphElement(comment))
+    comments.appendChild(createCommentDiv(comment))
+    comments.appendChild(document.createElement('br'));
   });
 }
 
-function createParagraphElement(text) {
-  const paragraphElement = document.createElement('p');
-  paragraphElement.innerText = text;
-  paragraphElement.className = "comment"
-  return paragraphElement;
+function createCommentDiv(commentObj) {
+  const commentDiv = document.createElement('div');
+  commentDiv.className = "comment";
+
+  const nameElement = document.createElement('h4');
+  nameElement.innerText = commentObj.name;
+  commentDiv.appendChild(nameElement);
+
+  const contentElement = document.createElement('p');
+  contentElement.innerText = commentObj.content;
+  commentDiv.appendChild(contentElement);
+
+  // TODO: do something useful with the time information
```

Comment:
You could maybe display the time when hovering over the comment? https://www.w3schools.com/css/css_tooltip.asp

Reply:
That could be cool! I might add that later.